packages/application-shell/src/utils/get-mc-api-url/get-mc-api-url.ts

```diff
@@ -1,55 +1,45 @@
-export interface ApplicationWindow extends Window {
-  app: {
-    servedByProxy: string | boolean;
-    mcApiUrl: string;
-  };
-}
+// NOTE: Copied from `@application-shell-connectors`.
+// TODO: Remove this version once the change in the connectors has been released.
```

Comment:
So the reason for this is that is this:

<img width="945" alt="image" src="https://user-images.githubusercontent.com/1110551/183455873-67e4c27c-037d-4294-b40f-9c74a877813d.png">

It seems that when we build/package the packages for attempting to install a starter template in isolation, there are some issues with cross-dependencies, maybe related to how packages are linked together.

To be honest, this is surprising and it means that either that particular setup is somehow wrong or it's not really possible to do that within a monorepo. I don't know at this point how to tackle that.
Definitely something to investigate but in the meantime I only have this solution.

PS: as a follow up, after merging/releasing this, we can be remove this again and the issue should be gone.